setup.py

```diff
@@ -1,183 +1,10 @@
-import argparse
-import copy
-import sysconfig
-import os
-import sys
-import traceback
+import setuptools
+import setuptools.command.build_ext
+import Cython.Build
 
-from setuptools import setup, Extension, find_packages
-from setuptools.command.build_ext import build_ext
+import numpy
 
-from pymoo.version import __version__
-
-# ---------------------------------------------------------------------------------------------------------
-# GENERAL
-# ---------------------------------------------------------------------------------------------------------
-
-
-__name__ = "pymoo"
-__author__ = "Julian Blank"
-__url__ = "https://pymoo.org"
-data = dict(
-    name=__name__,
-    version=__version__,
-    author=__author__,
-    url=__url__,
-    python_requires='>=3.10',
-    author_email="blankjul@outlook.com",
-    description="Multi-Objective Optimization in Python",
-    license='Apache License 2.0',
-    keywords="optimization",
-    packages=find_packages(include=['pymoo', 'pymoo.*']),
-    include_package_data=True,
-    exclude_package_data={
-        '': ['*.c', '*.cpp', '*.pyx'],
-    },
-    install_requires=['numpy>=1.15',
-                      'scipy>=1.1',
-                      'matplotlib>=3',
-                      'autograd>=1.4',
-                      'cma>=3.2.2',
-                      'alive-progress',
-                      'dill',
-                      'Deprecated'],
-    platforms='any',
-    classifiers=[
-        'Intended Audience :: Developers',
-        'Intended Audience :: Science/Research',
-        'Operating System :: OS Independent',
-        'License :: OSI Approved :: Apache Software License',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.10',
-        'Programming Language :: Python :: 3.11',
-        'Programming Language :: Python :: 3.12',
-        'Programming Language :: Python :: 3.13',
-        'Topic :: Scientific/Engineering',
-        'Topic :: Scientific/Engineering :: Artificial Intelligence',
-        'Topic :: Scientific/Engineering :: Mathematics'
-    ]
+setuptools.setup(
+    ext_modules=Cython.Build.cythonize("src/pymoo/cython/*.pyx"),
+    include_dirs=[numpy.get_include()],
 )
```

Comment:
All the package metadata is now in `pyproject.toml`. I'm very happy that we can make this setup.py so simple!